jwql/website/apps/jwql/bokeh_dashboard.py

```diff
@@ -44,6 +44,21 @@
 from jwql.website.apps.jwql.data_containers import build_table
 
 
+def disable_scientific_notation(figure):
+    """Disable y axis scientific notation.
+
+    Parameters
+    ----------
+    figure: bokeh figure object
+
+    Returns
+    -------
+    None
+    """
+    yaxis = figure.select(dict(type=Axis, layout="left"))[0]
+    yaxis.formatter.use_scientific = False
```

Comment:
Glad that this was relatively straightforward!

Reply:
Yeah, me too. Figured we might want to use it in other parts of the package so I made it a function.